src/ts/content/handlers.ts

```diff
@@ -38,13 +38,29 @@ function colorSelection() {
   }
 }
 
+const handleCheckIfApi = async () => {
+  const body = document.body;
+  const textContent = body.innerText || body.textContent;
+  const regexToTest = /API/;
```

Comment:
Good enough for now, maybe a simple comments to make sure we understand this is either temporary or requires intensive work in the future?
As a minor upgrade, I'd suggest `/\bAPI\b/`, so we will only capture word-like "API", and not part of existing strings.

Reply:
👍